package.json

```diff
@@ -47,14 +51,16 @@
     "postinstall": "if [ \"$ON_HEROKU\" ]; then npm install -g serve; fi"
   },
   "devDependencies": {
+    "@babel/core": "^7.16.0",
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "12.0.0",
     "babel-eslint": "^10.1.0",
-    "eslint": "^6.6.0",
+    "eslint": "7.12.0",
```

Comment:
CRA v4 uses eslint 7. I saw that you've went up to version 9. I tried to run 8 and 9, as well. And I think I did give it a good try. But I didn't manage it without errors. Probably it's because CRA v4 uses eslint 7. Don't know for sure, though.